src/visualizers/visualizerSpace.js

```diff
@@ -22,12 +21,13 @@ const PlayerAudioSource = function (player) {
   // source.connect(analyser);
   // analyser.connect(audioCtx.destination);
   const sampleAudioStream = function () {
-    analyser.getByteFrequencyData(dataArray);
+    if(!audioData.analyser) return;
+    audioData.analyser.getByteFrequencyData(audioData.dataArray);
     // calculate an overall volume value
     let total = 0;
     for (let i = 0; i < 50; i++) {
       // get the volume from the first 80 bins, else it gets too loud with treble
```

Comment:
The loop limit says `50` but this comment here suggests `80`. Is this intended?

Reply:
Yes I have tweaked it to test the best-suited value, can change later.

Reply:
That's totally fine, maybe just update the comment to reflect this test to be up-to-date.